research/model_description.ipynb

```diff
@@ -507,8 +516,175 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Examples\n",
-    "Create test pipeline and run it 10 epochs."
+    "## Inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To make predictions on new data first we load time and derivarive data as numpy arrays of shape (n_items, ) with dtype object (because arrays for every item have different length)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 121,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_log_data = np.load('../radial/tests/data/new_log.npz')\n",
```

Comment:
I think it will be better if you change path to'/path/to/data/file.npz' or smth like this.